bot/cogs/events.py

```diff
@@ -103,6 +104,30 @@ def mod_log(self) -> ModLog:
         resp = await response.json()
         return resp["data"]
 
+    async def has_active_mute(self, user_id: str) -> bool:
+        """
+        Check whether a user has any active mute infractions
+        """
+
+        response = await self.bot.http_session.get(
+            URLs.site_infractions_user.format(
+                user_id=user_id
+            ),
+            params={"hidden": "True"},
+            headers=self.headers
+        )
+        infraction_list = await response.json()
+
+        # Check for active mute infractions
+        if not infraction_list:
+            # Short circuit
+            return False
+
+        muted_check = any(
+            infraction["active"] for infraction in infraction_list if infraction["type"].lower() == "mute"
+        )
+        return muted_check
```

Comment:
How about renaming this to the something more explanatory like `is_muted` or just returning from `any` directly?